pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -510,6 +510,85 @@ Block.Component:
           minimum: 1
           nullable: true
           description: the number to label this block, `null` if it shouldn't be numbered
+Block.Activity:
+  allOf:
+    - $ref: '../pepper.yml#/components/schemas/Block'
+    - type: object
+      required:
+        - displayNumber
+        - renderHint
+        - activityCode
+        - allowMultiple
+      properties:
+        blockType:
+          type: string
+          default: ACTIVITY
+          enum:
+            - ACTIVITY
+        activityCode:
+          type: string
+        addButtonText:
+          type: string
+          nullable: true
+        allowMultiple:
+          type: boolean
+        renderHint:
+          type: string
+          description: |
+            give client an indication of how to display the the child activity instance. EMBEDDED suggests
+            to display the body of the activity instance contents in web page, MODAL suggests to display in
+            separate modal dialog.
+          enum:
+            - EMBEDDED
+            - MODAL
+        instances:
+          type: array
+          items:
+            $ref: '../pepper.yml#/components/schemas/Block.ActivityInstance'
+        displayNumber:
+          type: integer
+          minimum: 1
+          nullable: true
+          description: 'the number to label this block, `null` if it shouldn''t be numbered'
+Block.ActivityInstance:
+  allOf:
+    - $ref: '../pepper.yml#/components/schemas/Block'
+    - type: object
+      required:
+        - instanceGuid
+      properties:
+        blockType:
+          type: string
+          default: ACTIVITY_INSTANCE
+          enum:
+            - ACTIVITY_INSTANCE
+        instanceGuid:
+          type: string
+        readonly:
+          type: boolean
+          description: |
+            whether the activity instance is editable.
+        canDelete:
+          type: boolean
+          description: |
+            whether the activity instance can be deleted.
+        title:
+          type: string
+          nullable: true
+        subtitle:
+          type: string
+          nullable: true
+        subtitleSourceStableId:
+          type: string
+          nullable: true
+          description: |
+            the question stable id for answer that would provide the value for the
+            subtitle. Can be used by client to determine value of `subtitle`
+        displayNumber:
```

Comment:
Is the client going to be doing anything with this value other than displaying it (eg: performing arithmetic operations on it, appending text, etc.)? If not, the API may be better served by sending this as a display string- that way it can be expanded later by the server if we want to support non-integer schemes, without requiring an api revision.